pom.xml

```diff
@@ -950,26 +904,9 @@
         </file>
       </activation>
       <properties>
-        <java.level>8</java.level>
+        <java.level>11</java.level>
```

Comment:
No this must be reverted so long as https://github.com/jenkinsci/plugin-pom/blob/3325bb8e8a67c192b97fc6fe130265736d6480e1/pom.xml#L64

Reply:
It cannot be reverted while also accomplishing the purpose of this PR, which is stated in the PR description:

> Drop support for Java 8

The minimum Jenkins version cannot yet be updated because we do not know which version will drop support for Java 8, but the intention is that when we do we would update the minimum Jenkins version to that version.

Reply:
Right, I misunderstood the intent of this PR.